src/main/java/rx/internal/util/IndexedRingBuffer.java

```diff
@@ -47,6 +47,10 @@
  * @param <E>
  */
 public final class IndexedRingBuffer<E> implements Subscription {
+    private final ElementSection<E> elements = new ElementSection<E>();
+    private final IndexSection removed = new IndexSection();
+    /* package for unit testing */final AtomicInteger index = new AtomicInteger();
```

Comment:
That's fair. I still think it wouldn't hurt to just make one internally. Cleaner than comments and can still be picked up by IDEs anyway